src/planwise/client/projects/api.cljs

```diff
@@ -7,14 +7,14 @@
   (let [processed-filters (into {} (for [[k v] filters] [k (if (set? v) (apply vector v) v)]))]
     (if (seq (:type filters))
       processed-filters
-      ;; bogus facility type ID to reject all facilities
-      (assoc processed-filters :type ["0"]))))
+      (assoc processed-filters :type ""))))
 
 
-(defn load-project [id & handlers]
-  (GET
-    (str "/api/projects/" id)
-    (json-request {:id id} handlers)))
+(defn load-project [id with-data & handlers]
+  (let [url (str "/api/projects/" id)
+        params {:id id
+                :with (some-> with-data name)}]
```

Comment:
Do we need to repeat the `id` param in the request params, if it's included in the URL? I've actually done the same for the update, though.